PowerEditor/src/Parameters.cpp

```diff
@@ -4020,6 +4020,31 @@ void NppParameters::feedGUIParameters(TiXmlNode *node)
 			}
 		}
 
+		else if (!lstrcmp(nm, TEXT("SmartHighLightMode")))

```

Comment:
Seems like a bad habit to change it mid-way: I would say either refactor the entire file or stick to the methods already used there.